packages/baremetal-operator/charts/charts/baremetal-operator-crds/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
```

Comment:
Could you please explain why the chart is in both `packages/baremetal-operator-crds` and `packages/baremetal-operator/charts/charts/baremetal-operator-crds`?

This isn't the case for MetalLB for example.

Reply:
Honestly IDK...
This is the result from `make prepare`.
The diffs between both components are that:
1. Metallb is released chart until BarametalOperator not because it is part of the `Metal3` chart.
2. MetalLB is a root chart, until BaremetalOperator is a subchart (We have Metal3 -> BaremetalOperator -> BaremetalOperatorCRDs)

Reply:
I think you need to run `make clean` to get rid of this "working copy" according to the [development docs](https://github.com/suse-edge/charts/blob/main/docs/developing.md#making-changes-to-packages).